Client/Frontend/Home/HistoryHighlights/FxHomeHistoryHighlightsCollectionCell.swift

```diff
@@ -0,0 +1,241 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Storage
+
+struct HistoryHighlightsCollectionCellConstants {
```

Comment:
nit: `HistoryHighlightsCollectionCellConstants` -> `HistoryHighlightsCellConstants`

would be nice to shorten it but I am okay eitherways